go-controller/pkg/node/openflow_manager.go

```diff
@@ -266,8 +266,11 @@ func bootstrapOVSFlows() error {
 
 	var bridge string
 	var patchPort string
-	// patch-br-int-to-<bridge name>_<node>
-	r := regexp.MustCompile("^patch-(.*)_.*?-to-br-int$")
+	// This needs to work with:
+	// - default network: patch-<bridge name>_<node>-to-br-int
+	// but not with:
+	// - secondary network: patch-<bridge name>_<node>-to-br-int
+	r := regexp.MustCompile(fmt.Sprintf("^patch-([^_]*)_%s-to-br-int$", nodeName))
```

Comment:
this regexp isn't bulletproof cause you can have an OVS bridge with _ in it:

```
    Bridge br_blah
        Port br_blah
            Interface br_blah
                type: internal
    ovs_version: "3.2.2"
```

You should be able to use .*, since a kubernetes node name cannot have an underscore in the name.

Reply:
if I run that regex, it breaks with our UDN names. 

For instance, we would extract this bridge name - `br-ex_tenant-blue` - from the patch port `patch-br-ex_tenant-blue_multi-homing-worker-0.maiqueb.org-to-br-int`. 

Our UDN names will add another `_` separator between the bridge and the node name name ...

Reply:
Sorry I missed that the UDN networks have the node name in them and use underscore to separate the network. In that case we cannot use .*, but we also cannot then use underscore in our bridge names. What you propose is better than what we currently have, so lets merge it. In the future we can see if we can find a better mechanism of determining this port. Maybe add an external_id to the default patch port or something.

Reply:
100 %